packages/sui-studio/scripts/postinstall.js

```diff
@@ -0,0 +1,84 @@
+const fse = require('fs-extra')
+const fg = require('fast-glob')
+
+console.log(`Looking for test folder in ${process.env.INIT_CWD}`)
+
+if (fse.existsSync(`${process.env.INIT_CWD}/test`)) {
+  console.log('Test folder in place')
+  process.exit(0)
+}
+console.log(`Generating test folder in ${process.env.INIT_CWD}`)
+
+const components = fg
+  .sync([`${process.env.INIT_CWD}/components/**/src/index.js`])
+  .map(path =>
+    path
+      .replace(process.env.INIT_CWD + '/components/', '')
+      .replace('/src/index.js', '')
+  )
+
+const showError = msg => {
+  console.error(msg)
+  process.exit(1)
+}
+
+const writeFile = (path, body) => {
+  return new Promise((resolve, reject) => {
+    fse.outputFile(path, body, err => {
```

Comment:
fs-extra already returns a Promise so you don't need to wrap it in one.

Reply:
Copy and paste power!